bikram.md

```diff
@@ -0,0 +1,95 @@
+1. API Authentication:
+Every registered RESTful API should be assigned a unique Client ID and a Client Secret as a part of an HTTP header. The Client Secret should not be shared. Do not solely rely on a 'Client ID' for authentication.
```

Comment:
No. It's every RESTful API. 

So for every API there should be a specific Client ID and a Client Secret (ofc. mapped to an individual user)